src/core/Slider/Slider.tsx

```diff
@@ -12,6 +12,47 @@ import { getBoundedValue } from '../utils/common';
 import { Track } from './Track';
 import { Thumb } from './Thumb';
 
+// based on https://usehooks.com/useEventListener/
+function usePointerEventListener(
+  eventName: string,
+  handler: (event: PointerEvent) => void,
+  element: HTMLElement | Document | undefined,
+) {
+  // Create a ref that stores handler
+  const savedHandler = React.useRef<(event: PointerEvent) => void>();
+
+  // Update ref.current value if handler changes.
+  // This allows our effect below to always get latest handler ...
+  // ... without us needing to pass it in effect deps array ...
+  // ... and potentially cause effect to re-run every render.
```

Comment:
This comment can be shortened and combined with the comment above.

Reply:
comments combined